inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_non_max_suppression.cpp

```diff
@@ -85,31 +87,10 @@ class DynamicToStaticShapeNonMaxSuppression : public CommonTestUtils::TestsCommo
         const auto dims = std::make_shared<ngraph::opset3::Parameter>(ngraph::element::i64, ngraph::Shape{3});
         const auto dsr = std::make_shared<ngraph::vpu::op::DynamicShapeResolver>(scores, dims);
 
-        const auto node = std::make_shared<ngraph::opset3::NonMaxSuppression>(
+        const auto node = std::make_shared<ngraph::vpu::op::StaticShapeNonMaxSuppression>(
```

Comment:
I see there is small difference in test fixture for NMS v3 and NMS v4. I think it's reasonable to extract a super class to have only different methods in derived classes.